phplib/utils.php

```diff
@@ -58,10 +58,22 @@ static function fetch_json($hostname,$port,$protocol,$url) {
 
 
     static function deep_ksort(&$arr) {
```

Comment:
Honestly, I thought about this as well, but I was iterating fast and repeating the pattern of using cookies as I added features. I can definitely do this in a future update.

On a related note, I was considering consolidating all of the cookie values into a hash and setting a single cookie value from that. That will consolidate some more of the code, I think.